### DISCUSSION TITLE
Archaeological stratigraphy data

### DISCUSSION THREAD

**Discussion by ddomizia** [archeofoss/archeofoss2022#8] [Panel proposals]
Panel proposal

## Archaeological stratigraphy data.

![bldg-1-5-sequence](https://user-images.githubusercontent.com/39925492/163437687-68380297-6b92-4b36-9a79-7592680c73a1.png)

Proponents:
- **Stefano Costa**
  - Ministero della Cultura
  - [stefano.costa@beniculturali.it](mailto:stefano.costa@beniculturali.it)
- **Emanuel Demetrescu**
  - Consiglio Nazionale delle Ricerche
  - [emanuel.demetrescu@cnr.it](mailto:emanuel.demetrescu@cnr.it)

Stratigraphy data is one of the basic building blocks of archaeological research processes, without significant limits in chronology or geography. The nature of stratigraphy data is intrinsically simple and this has allowed the creation of many independent tools to archive, manage and most often visualize even complex stratigraphies without the need to manually draw complex graphs. However, despite the simplicity of the conceptual model behind those visualizations, little is available in terms of standardized data formats, and the existing formats are tied to specific software programs rather than neutral and open formats. Many bespoke archaeological information systems include their own implementation of a stratigraphic relationships database, without any interoperability. However, several open source tools are now available that deal with stratigraphy data, with a focus on analysis, format conversion, visualization, integration with archaeological information systems and specialized profiles for virtual reconstruction. Many of these tools rely on third party software like the well known Graphviz suite and have a low profile being targeted for command line usage, but most importantly the main common point is to keep raw data as their source of truth, either as files or in a database.

This panel aims at comparing current ways to work specifically with stratigraphy data using existing or newly proposed open source tools, with a focus on interoperability and standardization of procedures.

Data visualization examples in the form of Harris matrix or other graph-based approaches are welcome, as long as they are based on open source software, to be released as literate programming notebooks or source code along with the final result.

Bibliography
- Costa, Stefano. 2019. “Una proposta di standard per l’archiviazione e la condivisione di dati stratigrafici.” Archeologia e Calcolatori 30 (January): 459–62. https://doi.org/10.19282/ac.30.2019.29.
- Demetrescu, Emanuel, and Daniele Ferdani. 2021. “From Field Archaeology to Virtual Reconstruction: A Five Steps Method Using the Extended Matrix.” Applied Sciences 11 (11): 5206. https://doi.org/10.3390/app11115206.
- Dye, Thomas S., and Caitlin E. Buck. 2015. “Archaeological Sequence Diagrams and Bayesian Chronological Models.” Journal of Archaeological Science 63 (Supplement C): 84–93. https://doi.org/10.1016/j.jas.2015.08.008.
- Moody, Bryony, Tom Dye, Keith May, Holly Wright, and Caitlin Buck. 2021. “Digital Chronological Data Reuse in Archaeology: Three Case Studies with Varying Purposes and Perspectives.” Journal of Archaeological Science: Reports 40 (December): 103188. https://doi.org/10.1016/j.jasrep.2021.103188.

**Comment by steko** [comment]
Other official proponents of the panel, who could not fit in the standard template, are:

Thomas S. Dye
Department of Anthropology, University of Hawai’i at M ̄ anoa, Honolulu, Hawaii
tsd@tsdye.online

Luca Mandolesi
adArte Srl
https://www.linkedin.com/in/luca-mandolesi-772b2815

Enzo Cocca
adArte Srl
enzo.ccc@gmail.com

Bryony Moody
Statistics Research Group, University of Sheffield
bcmoody1@sheffield.ac.uk

**Comment by benducke** [comment]
A classic topic that is still relevant. I wonder if this could be merged with another session: https://github.com/archeofoss/archeofoss2022/discussions/7
